cockpit/src/subscriptions-view.jsx

```diff
@@ -99,18 +99,29 @@ class InstalledProducts extends React.Component {
         let entries = this.props.products.map(function (itm) {
             let subscribed;
             let status_text;
+            let label_status_text;
             let start_date_text;
             let end_date_text;
             let body;
             let columns;
 
             if (itm.status === 'subscribed') {
-                subscribed = true;
-                status_text = _("Subscribed");
-
+                subscribed = "green";
+                label_status_text = _("Subscribed");
+                status_text = label_status_text;
+            } else if (itm.status === 'partially_subscribed') {
+                subscribed = "orange";
+                label_status_text = _("Partially subscribed");
+                status_text = label_status_text + ' (' + itm.status_details.join(',') + ')';
```

Comment:
this is a string puzzle, which makes things harder to translate and fit for the various languages, I'd suggest something like:
```js
  status_text = cockpit.format(_("Partially subscribed ($0)"),
                               itm.status_details.join(', '));
```